### PR TITLE
Fix for EditorManager(1) and MainViewManager(4) test failures

### PR DESCRIPTION
**Pull request by swmitra**
This PR resolves EditorManager and MainViewManager test failures. 

Ping @MarcelGerber @nethip @petetnt for review.

### CODE REVIEW

File: test/spec/EditorManager-test.js [hunk 1]
```diff
@@ -85,14 +85,21 @@ define(function (require, exports, module) {
...
                 EditorManager.openDocument(testDoc, anotherPane);
+
                 expect(pane.addView).not.toHaveBeenCalled();
                 expect(pane.showView).not.toHaveBeenCalled();
-                expect(anotherPane.addView).toHaveBeenCalled();
-                expect(anotherPane.addView).not.toHaveBeenCalledWith(editor);
-                expect(anotherPane.showView).toHaveBeenCalled();
-                expect(anotherPane.showView).not.toHaveBeenCalledWith(editor);
+                expect(anotherPane.addView).toHaveBeenCalledWith(editor);
+                expect(anotherPane.addView).toHaveBeenCalledWith(editor);
```
Comment (MarcelGerber) [inline test/spec/EditorManager-test.js]: I don't think this line is meant to be duplicated? (Same below with `expect(pane.showView).not.toHaveBeenCalledWith(editor);`)

File: test/spec/MainViewManager-test.js [hunk 2]
```diff
@@ -877,7 +880,8 @@ define(function (require, exports, module) {
                     expect(MainViewManager._getPaneIdForPath(getFileObject("test.html").fullPath)).toEqual("second-pane");
                 });
             });
-            it("should not add files if they exist in other panes", function () {
+            // This test is no longer valid with split view for same doc
```
Comment (petetnt) [inline test/spec/MainViewManager-test.js]: I think this can be removed altogether instead of commenting it out, right?

**Comment by petetnt** [comment]
All tests pass so sans the two small things above LGTM

**Comment by swmitra** [comment]
Thanks a lot @MarcelGerber @petetnt for the quick review :+1: 
@nethip @MarcelGerber you can merge the changes if everything is fine :smile:

**Comment by nethip** [comment]
LGTM. Thanks @swmitra for fixing this.

**Pull request merged**